earth_enterprise/src/fusion/autoingest/sysman/DependentStateTree.cpp

```diff
@@ -168,7 +171,7 @@ void DependentStateTreeFactory::FillInVertex(
   if (vertices[name].includeConnections) {
     // In some cases, assets in the dependent tree don't need their inputs and
```

Comment:
Minor: this comment is slightly out of date. Something like "In some cases, assets don't need all their inputs and children to calculate their states.